docs/source/models/causes/neonatal/index.rst

```diff
@@ -192,29 +238,26 @@ calculations.
       - Definition
       - Value or source
       - Note
-    * - mr
-      - neonatal mortality risk per live birth
-      - deaths_c380 / live_births
-      - The value of mr is a probabiity in [0,1]. Denominator includes
-        live births only.
-    * - deaths_c380
-      - cause-specific deaths due to neonatal disorders
-      - codcorrect
-      - just for neonatal age groups
-    * - live_births
-      - birth rate (live births only)
-      - sum(ASFR*population)
-      - Units are total live births  per person-year
-    * - ASFR
-      - Age-specific fertility rate
-      - get_covariate_estimates: coviarate_id=13
-      - Assume lognormal distribution of uncertainty. Units in GBD are
-        live births per person, or equivalently, per person-year.
-    * - population
-      - average population in a given year
-      - get_population
-      - Specific to age/sex/location/year demographic group. Numerically
-        equal to person-time for the year.
+    * - ACMR
+      - all-cause mortality rate (per person year)
+      - GBD
+      -
+    * - PAF_LBWSG
+      - population attributable fraction of all-cause mortality for low birth weight and short gestation
+      - Computed to be consistent with GBD
```

Comment:
What does this mean? I.e., in what way are we trying to be consistent with GBD? Presumably we want to match GBD's ACMR, but we don't really need to know that to compute the LBWSG PAF. We just need to compute PAF = 1 - 1 / E(RR) from the interpolated relative risk function.

Reply:
I will make that clearer